app/models/bill.rb

```diff
@@ -0,0 +1,4 @@
+class Bill < ApplicationRecord
+  belongs_to :from_user, class_name: User.name, foreign_key: "user_id"
```

Comment:
```suggestion
  belongs_to :from_user, class_name: User.name, foreign_key: :user_id
```
tương tự cho các chỗ khác